README.md

```diff
@@ -176,14 +176,73 @@ let nftType = nft.getType()
+| `Royalties` | Wrapper for multiple Royalty views.        | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L266-L290) |
+| `Media`     |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L311-L328)|
+| `Medias`    |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L330-L340)|
+| `License`   |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L353-L362)|
+| `ExternalURL`|                                           | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L375-L385)|
+| `NFTCollectionData` | Provides storage and retrieval information of an NFT | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L398-L455) |
+| `NFTCollectionDisplay` | Returns the basic representation of an NFT's Collection.  | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L468-L507) |
+| `Rarity`   |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L523-L550)|
+| `Trait`    |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L563-L590)|
+| `Traits`   |                                            | Implemented | [MetadataViews.cdc](https://github.com/onflow/flow-nft/blob/master/contracts/MetadataViews.cdc#L592-L605)|
+
+## Allways prefer wrappers over single views
+
+When exposing a view that could have multiple ocurrences on a single NFT, such as `Edition`, `Royalty`, `Media` or `Trait` the wrapper view should allways be exposed, even if there is a single occurrence.
+
+When resolving the view, the wrapper view should be the returned value, instead of returning the single view or just an array of several ocurrences of the view
+
+### Example
+
+#### Prefered
+
+```cadence
+pub fun resolveView(_ view: Type): AnyStruct? {
+    switch view {
+        case Type<MetadataViews.Editions>():
+            let editionInfo = MetadataViews.Edition(name: "Example NFT Edition", number: self.id, max: nil)
+            let editionList: [MetadataViews.Edition] = [editionInfo]
+            return MetadataViews.Editions(
+                editionList
+            )
+    }
+}
+```
+
+#### To be avoided
+
+```cadence
+pub fun resolveView(_ view: Type): AnyStruct? {
```

Comment:
```suggestion
// `resolveView` should always return the same type that was passed to it as an argument, so this is improper usage because it returns `Edition` instead of `Editions`.
pub fun resolveView(_ view: Type): AnyStruct? {
```